server/app/views/admin/programs/ProgramBlockEditView.java

```diff
@@ -616,6 +616,19 @@ private DivTag renderQuestion(
               blockDefinition.id(),
               questionDefinition,
               canRemove));
+    } else {
+      // For each toggle, use a label instead in the read only view
+      if (maybeAddressCorrectionEnabledToggle.isPresent()) {
+        String label =
+            addressCorrectionEnabled
+                ? "Address correction: enabled"
```

Comment:
**suggestion (non-blocking):** Optionally can pull this common part out and just use the ternary for the enabled / disabled piece.

Reply:
Done